device_services/generic/chargepoint.md

```diff
@@ -8,38 +8,55 @@ Chargepoint service is used to represent EV chargers.
+| in   | cmd.current_session.set_current | int        |                                                                     | Sets the offered current for the ongoing session in `A`, must be an integer between `6` and a set `max_current` value. |
+| in   | cmd.current_session.get_report  | null       |                                                                     | Requests energy consumed during the current session and optionally additional characteristics.                         |
+| out  | evt.current_session.report      | float      | `previous_session`, `started_at`,  `finished_at`, `offered_current` | Reports energy consumed during the current session in `kWh` and optionally additional characteristics.                 |
+| -    |                                 |            |                                                                     |                                                                                                                        |
+| in   | cmd.max_current.set             | int        |                                                                     | Sets the maximum offered current in `A`, must be an integer between `6` and `sup_max_current` service property value.  |
+| in   | cmd.max_current.get_report      | null       |                                                                     | Requests the maximum offered energy.                                                                                   |
+| out  | evt.max_current.report          | int        |                                                                     | Reports the maximum offered current in `A`, this is effectively a static load balancing value.                         |
 
 ## Interface properties
 
-| Name            | Example  | Required | Description                                                                            |
-|-----------------|----------|----------|----------------------------------------------------------------------------------------|
-| `charging_mode` | `"slow"` | No       | One of charging modes defined in [`sup_charging_modes`](#service-properties) property. |
+| Name               | Example                       | Required | Description                                                                                      |
+|--------------------|-------------------------------|----------|--------------------------------------------------------------------------------------------------|
+| `charging_mode`    | `"slow"`                      | No       | One of charging modes defined in [`sup_charging_modes`](#service-properties) property.           |
+| `previous_session` | `"2.5"`                       | No       | Reports energy consumed during previous session in `kWh`.                                        |
+| `started_at`       | `"2022-09-01T08:00:00Z02:00"` | No       | Time of current session start in RFC3339 format.                                                 |
```

Comment:
Present only if the current session has started. ? or this is for the previous session when finished ?

Reply:
Yes. It this is first time the charger is connected to the system it can be empty, otherwise there will always be a current session, even if its finished.